pyworkflow/gui/gui.py

```diff
@@ -39,8 +39,8 @@
 FONT_NORMAL = 'fontNormal'
 FONT_BOLD = 'fontBold'
 FONT_BIG = 'fontBig'
-cfgFontName = os.environ.get('SCIPION_FONT_NAME', "Helvetica")
-cfgFontSize = int(os.environ.get('SCIPION_FONT_SIZE', 10))
+cfgFontName = pwConfig.SCIPION_FONT_NAME
```

Comment:
Why not keeping consisting usage as pw.Config?

Reply:
sure.